### PR TITLE
Adds shuttle docks to runtimestation

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Runtimestation: Added shuttle docking ports, a comms console, cargo bay and syndicate/centcom IDs.
/:cl:

Right now there is no test map for shuttles; using the regular maps gets pretty annoying.
I added all docks while trying to keep the layout compact - arrivals, departures, pods, cargo, gulag, mining, ferry, aux/whiteship. 

Also, added syndie/centcom IDs for ID/access testing and some light fixtures to the western construction area.

[Screenshot](https://i.imgur.com/iWVscOH.png)

### CODE REVIEW

File: _maps/map_files/debug/runtimestation.dmm [hunk 1]
```diff
@@ -1295,356 +1361,8090 @@
...
+/turf/closed/wall/r_wall,
+/area/hallway/secondary/entry)
+"gf" = (
+/obj/effect/spawner/structure/window/reinforced,
+/turf/open/floor/plating,
+/area/hallway/secondary/exit/departure_lounge)
+"gg" = (
+/obj/structure/sign/directions/supply{
+	dir = 4;
+	icon_state = "direction_supply";
```
Comment (ShizCalev) [inline _maps/map_files/debug/runtimestation.dmm]: dv

File: _maps/map_files/debug/runtimestation.dmm [hunk 2]
```diff
@@ -1295,356 +1361,8090 @@
...
+	pixel_x = 32
+	},
+/obj/structure/sign/directions/engineering{
+	dir = 1;
+	icon_state = "direction_eng";
```
Comment (ShizCalev) [inline _maps/map_files/debug/runtimestation.dmm]: dv

File: _maps/map_files/debug/runtimestation.dmm [hunk 3]
```diff
@@ -1295,356 +1361,8090 @@
...
+/area/hallway/secondary/exit/departure_lounge)
+"fX" = (
+/obj/structure/sign/directions/supply{
+	dir = 4;
+	icon_state = "direction_supply";
+	pixel_x = -32
+	},
+/obj/structure/sign/directions/engineering{
+	dir = 1;
+	icon_state = "direction_eng";
```
Comment (ShizCalev) [inline _maps/map_files/debug/runtimestation.dmm]: dv

File: _maps/map_files/debug/runtimestation.dmm [hunk 4]
```diff
@@ -1295,356 +1361,8090 @@
...
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden{
+	dir = 9
+	},
+/turf/open/floor/plasteel,
+/area/hallway/secondary/exit/departure_lounge)
+"fX" = (
+/obj/structure/sign/directions/supply{
+	dir = 4;
+	icon_state = "direction_supply";
```
Comment (ShizCalev) [inline _maps/map_files/debug/runtimestation.dmm]: dv

File: _maps/map_files/debug/runtimestation.dmm [hunk 5]
```diff
@@ -1295,356 +1361,8090 @@
...
+/turf/open/floor/plasteel,
+/area/quartermaster/storage)
+"fp" = (
+/obj/docking_port/stationary{
+	dheight = 0;
+	dir = 2;
+	dwidth = 9;
+	height = 25;
+	id = "emergency_home";
+	name = "MetaStation emergency evac bay";
```
Comment (ShizCalev) [inline _maps/map_files/debug/runtimestation.dmm]: if you'd be so kind, please rename this

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/debug/runtimestation.dmm (hunk 5 above)
Comment (81Denton) [reply _maps/map_files/debug/runtimestation.dmm]: Ok, done!

**Review by ShizCalev — DISMISSED** [review]

**Comment by SpaceManiac** [comment]
Can you add fans to the escape pod bay as in #38523, so we're not knowingly causing roundstart ATs? Otherwise looks good.

**Comment by 81Denton** [comment]
Yeah, I'll ping you once it's done.

**Comment by 81Denton** [comment]
@SpaceManiac Done!

**Pull request merged**